sdk/java/src/main/java/sawtooth/sdk/client/Signing.java

```diff
@@ -22,7 +36,7 @@ public static ECKey generatePrivateKey(SecureRandom random) {
   }
 
   public static String getPublicKey(ECKey privateKey) {
-    return "03" + privateKey.getPublicKeyAsHex();
+    return privateKey.getPublicKeyAsHex();
```

Comment:
Could you explain why you removed the "03" from the returned value? It's hard to approve this without understanding why this is a good thing to remove!